docs/assets/css/copy-button.css

```diff
@@ -0,0 +1,27 @@
+.copy-button {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 5px 10px;
+    font-size: 14px;
+    cursor: pointer;
+    border: 1px solid #009E7F;
+    border-radius: 3px;
+    background-color: #009E7F;
+    color: #fff;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
Replace the font-family with the one that is already in use.

Reply:
okay see for all that you can suggest,
 anything you else you find  that needs attention.